tests fail: Error: Symbol 'source_dir' at (1) has no IMPLICIT type

Tests run for while until this happens:
```
ok 1 - 
      LINK.F build/t400-qfunction-f
/usr/ports/math/libceed/work/libCEED-0.12.0/tests/t400-qfunction-f.f90:30:16:

   30 |      &SOURCE_DIR&
      |                1
Error: Symbol 'source_dir' at (1) has no IMPLICIT type
gmake: *** [Makefile:571: build/t400-qfunction-f] Error 1
*** Error code 1
```

Version: 0.12.0
gfortran13
FreeBSD 14.1

The current master revision e3ae47f6 fails in a different way:
```
ok 1 - 
      LINK.c build/t406-qfunction
In file included from /usr/ports/math/libceed/work/libCEED-0.12.0-763-ge3ae47f6/tests/t406-qfunction.c:4:
/usr/ports/math/libceed/work/libCEED-0.12.0-763-ge3ae47f6/tests/t406-qfunction.h:22:10: fatal error: '../../libCEED/tests/t406-qfunction-helper.h' file not found
   22 | #include "../../libCEED/tests/t406-qfunction-helper.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

```
$ find . -name t406-qfunction-helper.h
./work/libCEED-0.12.0-763-ge3ae47f6/tests/t406-qfunction-helper.h
```

--- a/tests/t406-qfunction.h
+++ b/tests/t406-qfunction.h
@@ -19,7 +19,7 @@
 // Also test include path with "/../"
 #include "../tests/t406-qfunction-helper.h"
 // Also test include path with "/../../"
-#include "../../libCEED/tests/t406-qfunction-helper.h"
+#include "output/../../tests/t406-qfunction-helper.h"
 #  include "t406-qfunction-scales.h"
 // clang-format on
 

